2022-12-09 13:09:29.1386 DEBUG Инициализация программы
2022-12-09 13:09:29.4788 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 13:09:31.4166 ERROR Ошибка MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.Net.Sockets.SocketException (0x80004005): Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение 127.0.0.1:3306
   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   в System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
   в MySql.Data.MySqlClient.NativeDriver.<>c.<Open>b__40_0(Exception ex)
   в System.AggregateException.Handle(Func`2 predicate)
   в MySql.Data.MySqlClient.NativeDriver.Open()
   в MySql.Data.MySqlClient.Driver.Open()
   в MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   в MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   в MySql.Data.MySqlClient.MySqlPool.GetConnection()
   в MySql.Data.MySqlClient.MySqlConnection.Open()
   в System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   в Liftais.MainWindow.SingIn_Click(Object sender, RoutedEventArgs y) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\MainWindow.xaml.cs:строка 55
2022-12-09 13:09:56.0600 DEBUG Инициализация программы
2022-12-09 13:09:56.0600 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 13:09:58.8549 DEBUG Успешное подключение 
Аккаунт: admin
2022-12-09 13:09:58.8549 DEBUG Попытка входа
2022-12-09 13:09:58.8549 INFO Успешный вход
2022-12-09 14:05:12.2588 DEBUG Инициализация программы
2022-12-09 14:05:12.4764 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:05:19.3890 DEBUG Успешное подключение 
Аккаунт: admin
2022-12-09 14:05:19.3890 DEBUG Попытка входа
2022-12-09 14:05:19.3890 INFO Успешный вход
2022-12-09 14:46:21.8868 DEBUG Инициализация программы
2022-12-09 14:46:22.3633 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:46:23.1194 ERROR Ошибка MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.Net.Sockets.SocketException (0x80004005): Сделана попытка выполнить операцию на сокете при отключенной сети 192.168.88.54:3306
   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   в System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
   в MySql.Data.MySqlClient.NativeDriver.<>c.<Open>b__40_0(Exception ex)
   в System.AggregateException.Handle(Func`2 predicate)
   в MySql.Data.MySqlClient.NativeDriver.Open()
   в MySql.Data.MySqlClient.Driver.Open()
   в MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   в MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   в MySql.Data.MySqlClient.MySqlPool.GetConnection()
   в MySql.Data.MySqlClient.MySqlConnection.Open()
   в System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   в Liftais.MainWindow.SingIn_Click(Object sender, RoutedEventArgs y) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\MainWindow.xaml.cs:строка 55
2022-12-09 14:46:30.8820 DEBUG Инициализация программы
2022-12-09 14:46:30.8820 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:46:30.8840 ERROR Ошибка MySql.Data.MySqlClient.MySqlException (0x80004005): Unable to connect to any of the specified MySQL hosts. ---> System.Net.Sockets.SocketException (0x80004005): Сделана попытка выполнить операцию на сокете при отключенной сети 192.168.88.54:3306
   в System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   в System.Net.Sockets.TcpClient.EndConnect(IAsyncResult asyncResult)
   в System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
   в MySql.Data.MySqlClient.NativeDriver.<>c.<Open>b__40_0(Exception ex)
   в System.AggregateException.Handle(Func`2 predicate)
   в MySql.Data.MySqlClient.NativeDriver.Open()
   в MySql.Data.MySqlClient.Driver.Open()
   в MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   в MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   в MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   в MySql.Data.MySqlClient.MySqlPool.GetConnection()
   в MySql.Data.MySqlClient.MySqlConnection.Open()
   в System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   в System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   в Liftais.MainWindow.SingIn_Click(Object sender, RoutedEventArgs y) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\MainWindow.xaml.cs:строка 55
2022-12-09 14:47:46.8571 DEBUG Инициализация программы
2022-12-09 14:47:46.9131 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:47:50.6360 DEBUG Успешное подключение 
Аккаунт: admin
2022-12-09 14:47:50.6360 DEBUG Попытка входа
2022-12-09 14:47:50.6360 INFO Успешный вход
2022-12-09 14:49:35.0664 DEBUG Инициализация программы
2022-12-09 14:49:35.0664 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:49:35.6372 DEBUG Успешное подключение 
Аккаунт: Pankratov
2022-12-09 14:49:35.6372 DEBUG Попытка входа
2022-12-09 14:49:35.6372 INFO Успешный вход
2022-12-09 14:53:40.0458 DEBUG Инициализация программы
2022-12-09 14:53:40.1515 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 14:53:41.7280 DEBUG Успешное подключение 
Аккаунт: admin
2022-12-09 14:53:41.7280 DEBUG Попытка входа
2022-12-09 14:53:41.7280 INFO Успешный вход
2022-12-09 15:29:16.0012 DEBUG Инициализация программы
2022-12-09 15:29:16.0012 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 15:29:21.2415 DEBUG Успешное подключение 
Аккаунт: admin
2022-12-09 15:29:21.2415 DEBUG Попытка входа
2022-12-09 15:29:23.1975 INFO Неправильный логин или пароль
Аккаунт: admin
2022-12-09 16:03:49.7873 DEBUG Инициализация программы
2022-12-09 16:03:49.8423 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 16:03:56.3653 DEBUG Успешное подключение 
Аккаунт: Pankratov
2022-12-09 16:03:56.3653 DEBUG Попытка входа
2022-12-09 16:03:56.3653 INFO Успешный вход
2022-12-09 16:20:30.4970 DEBUG Инициализация программы
2022-12-09 16:20:30.5608 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 16:20:31.5526 DEBUG Успешное подключение 
Аккаунт: Pankratov
2022-12-09 16:20:31.5526 DEBUG Попытка входа
2022-12-09 16:20:31.5526 INFO Успешный вход
2022-12-09 16:38:21.4381 DEBUG Инициализация программы
2022-12-09 16:38:21.5079 DEBUG Выполнение запроса Sql + Подключение к Базе данных
2022-12-09 16:38:22.8936 DEBUG Успешное подключение 
Аккаунт: Жавохир
2022-12-09 16:38:22.8936 DEBUG Попытка входа
2022-12-09 16:38:22.8936 INFO Успешный вход
2022-12-09 18:03:43.7503 ERROR Ошибка создания записи в модуле ПосетителиMySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '880' for key 'id_visiter'
   в MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   в MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   в MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteNonQuery()
   в Liftais.Ais.cr_note_visiter_btn_Click(Object sender, RoutedEventArgs e) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\Ais.xaml.cs:строка 1227
2022-12-09 18:03:47.6906 ERROR Ошибка создания записи в модуле ПосетителиMySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '880' for key 'id_visiter'
   в MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   в MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   в MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteNonQuery()
   в Liftais.Ais.cr_note_visiter_btn_Click(Object sender, RoutedEventArgs e) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\Ais.xaml.cs:строка 1227
2022-12-09 18:04:08.7161 ERROR Ошибка создания записи в модуле ПосетителиMySql.Data.MySqlClient.MySqlException (0x80004005): Duplicate entry '880' for key 'id_visiter'
   в MySql.Data.MySqlClient.MySqlStream.ReadPacket()
   в MySql.Data.MySqlClient.NativeDriver.GetResult(Int32& affectedRow, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.GetResult(Int32 statementId, Int32& affectedRows, Int64& insertedId)
   в MySql.Data.MySqlClient.Driver.NextResult(Int32 statementId, Boolean force)
   в MySql.Data.MySqlClient.MySqlDataReader.NextResult()
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteReader(CommandBehavior behavior)
   в MySql.Data.MySqlClient.MySqlCommand.ExecuteNonQuery()
   в Liftais.Ais.cr_note_visiter_btn_Click(Object sender, RoutedEventArgs e) в C:\Users\Семён\Documents\GitHub\Practice_Lift1\Liftais\Liftais\Ais.xaml.cs:строка 1227
